### ISSUE TITLE
ExperimentDataDictionary

### ISSUE THREAD

**Issue by Nick-G-Horvath** [varnerlab/Kwatee-CellFreeModel-Plugin#1]
We should generate `DataFile.jl` such that it contains the experimental data dictionary.

**Comment by Nick-G-Horvath** [comment]
We should generate all files to use relative paths.

**Comment by Nick-G-Horvath** [comment]
In `Objective.jl`, the `reshape` on line 10 has a `16` hard coded. Instead of `tmp_array = reshape(tmp,16,2);`, perhaps `num_row = convert(Int64,length(tmp)/2); tmp_array = reshape(tmp,num_row,2);`. The same is true of `SearchRC.jl`, `SearchK.jl`, and `SearchIC.jl`.

**Comment by varnerlab** [comment]
I agree with **almost** all of the comments with the exception of `Objective,jl`. `Objective,jl` is **not** generated by the plugin (this was hand coded by me). In the future, we would like to generate all the simulation files, and associated objective functions (similar to the BCX package in the old UNIVERSAL system). However, I haven't had time to think about this yet.

**Action items**
- I'll update the delegate method that generates `DataFile.jl` to use relative paths.
-  I'll update the README.md file to let users know what files gets generated, and where they are put. Also we should warn them that all files are overwritten when regenerated.
